res/cmake/def.cmake

```diff
@@ -16,7 +16,7 @@ endif(withIPO)
 
 set (PHARE_WITH_CCACHE FALSE)
 if(devMode) # -DdevMode=ON
-  set(testDuringBuild ON)
+    #set(testDuringBuild ON)
```

Comment:
this is enabled on CI even without this, so rm or uncomment?

Reply:
ok deleted